library/patient.inc

```diff
@@ -77,7 +77,7 @@ function getInsuranceProviders() {
     $returnval = array();
 
     if (true) {
-        $sql = "select name, id from insurance_companies order by name, id";
+        $sql = "select name, id from insurance_companies where inactive != 1 order by name, id";
         $rez = sqlStatement($sql);
         for($iter=0; $row=sqlFetchArray($rez); $iter++) {
```

Comment:
The getInsuranceProvidersExtra() function below this also needs to be modified to only return the active insurance companies.